src/services/Store/store/account.slice.ts

```diff
@@ -142,13 +143,23 @@ export const getAccounts = createSelector([getAppState], (s) => {
   const accounts = s.accounts;
   return accounts?.map((a) => ({
     ...a,
-    transactions: a.transactions?.map((t) => ({
-      ...t,
-      value: EthersBN.from(t.value),
-      gasLimit: EthersBN.from(t.gasLimit),
-      gasPrice: EthersBN.from(t.gasPrice),
-      gasUsed: t.gasUsed && EthersBN.from(t.gasUsed)
-    }))
+    transactions: a.transactions?.map((t) => {
+      const gas = isType2Receipt(t)
+        ? {
+            maxFeePerGas: EthersBN.from(t.maxFeePerGas),
+            maxPriorityFeePerGas: EthersBN.from(t.maxPriorityFeePerGas)
+          }
+        : { gasPrice: EthersBN.from(t.gasPrice) };
+
+      return {
+        ...t,
+        value: EthersBN.from(t.value),
+        gasLimit: EthersBN.from(t.gasLimit),
+        ...gas,
+        gasUsed: t.gasUsed && EthersBN.from(t.gasUsed),
+        nonce: EthersBN.from(t.nonce)
```

Comment:
`bigify`?

Reply:
We use Ethers BNs for user transactions too. Out of scope imo.